apps/desktop/src/lib/components/Board.svelte

```diff
@@ -83,9 +93,9 @@
 		class="board"
 		role="group"
 		data-tauri-drag-region
-		on:drop={(e) => {
+		ondrop={(e) => {
 			e.preventDefault();
-			if (!dragged) {
+			if (!dragged || !sortedBranches) {
```

Comment:
Is this a migration change or a bug fix?

Reply:
Bug fix. Just an extra guard I added to avoid situations where `sortedBranches` could be `undefined`